rbi/core/unbound_method.rbi

```diff
@@ -140,6 +140,10 @@ class UnboundMethod
   sig {params(recv: BasicObject, args: T.untyped, blk: T.untyped).returns(T.untyped)}
   def bind_call(recv, *args, &blk); end
 
+  # for our experimental procc patch
```

Comment:
Should we put an RBI for this in Stripe's codebase, instead of the payload that we ship to all users of Sorbet?

This will mean that `umethod.bind_cal|` will show `bind_call_experimental_optimized` in the completion results even for non-Stripe users.

Reply:
👍